scripts/test-ssr/src/utils/getExportFromFile.ts

```diff
@@ -91,7 +91,7 @@ export async function getExportFromFile(distDir: string, filename: string): Prom
     },
   };
 
-  await Babel.transformAsync(sourceCode, {
+  await Babel.transformSync(sourceCode, {
```

Comment:
why this change ? if we need sync isnt that await redundant ?

Reply:
Good catch, I reverted it